### DISCUSSION TITLE
Bluetooth on LilyGO LoRa32 v2.1 board connects to MacOS but cant send/recieve

### DISCUSSION THREAD

**Discussion by KnoxMajor** [markqvist/Reticulum#546] [Help & Questions]
I have this problem on the two boards I have created.
- Both are the board from the unsigned.io RNode build recipe: **LilyGO LoRa32 v2.1**
- Both boards work perfectly when wired (works on two different MacOS machines)
- Both connect with Bluetooth and show as connected and active (on two different MacOS machines)
- Both light up when announces are made (the green LED built into the board)
- However, both won't send/receive

**rnstatus**:
```
 Shared Instance[37428]
    Status    : Up
    Serving   : 1 program
    Rate      : 1.00 Gbps
    Traffic   : 401 B↑
                696 B↓

 RNodeInterface[RNode LoRa Interface]
    Status    : Down
    Mode      : Full
    Rate      : 3.12 kbps
    Airtime   : 7.43% (15s), 0.06% (1h)
    Ch.Load   : 23.17% (15s), 0.35% (1h)
    Traffic   : 412 B↑
                418 B↓

 RNodeInterface[RNode LoRa Interface (Bluetooth)]
    Status    : Up
    Mode      : Full
    Rate      : 3.12 kbps
    Airtime   : 0.0% (15s), 0.03% (1h)
    Ch.Load   : 0.0% (15s), 0.14% (1h)
    Traffic   : 1.35 KB↑
                412 B↓
```
**config**:
```
    [[RNode LoRa Interface (Bluetooth)]]
    type = RNodeInterface

    # Enable interface if you want use it!
    enabled = yes

    # Serial port for the device
    port = /dev/cu.RNodeAC3C

    # Set frequency to 867.2 MHz
    frequency = 867200000

    # Set LoRa bandwidth to 125 KHz
    bandwidth = 125000

    # Set TX power to 7 dBm (5 mW)
    txpower = 7

    # Select spreading factor 8. Valid 
    # range is 7 through 12, with 7
    # being the fastest and 12 having
    # the longest range.
    spreadingfactor = 8

    # Select coding rate 5. Valid range
    # is 5 throough 8, with 5 being the
    # fastest, and 8 the longest range.
    codingrate = 5

    # You can configure the RNode to send
    # out identification on the channel with
    # a set interval by configuring the
    # following two parameters. The trans-
    # ceiver will only ID if the set
    # interval has elapsed since it's last
    # actual transmission. The interval is
    # configured in seconds.
    # This option is commented out and not
    # used by default.
    # id_callsign = baron
    # id_interval = 600

    # For certain homebrew RNode interfaces
    # with low amounts of RAM, using packet
    # flow control can be useful. By default
    # it is disabled.
    flow_control = False
```

**Comment by KnoxMajor** [comment]
This may be the wrong place to post this, @markqvist feel free to point me in the right direction if so!
Also, thank you so much for all the incredible work you have created-- I have loved learning and using it!

**Reply by markqvist** [reply]
Hmm, that is odd, everything looks good. Could you add `loglevel = 7` to your config on both sides and post the log output from both sides when sending an announce?

**Reply by KnoxMajor** [reply]
After testing further (sorry for the delay), I discovered that this happens on my Mac machines instead of any other OS. I blame it on something finicky with Mac peripherals themselves, I also have older machines so it may be stable on newer Mac machines.

Nonetheless, this can be closed, I got it working on other OS machines. I will open an issue if I find an identifiable and addressable 'issue' later on.
